binder/src/main/java/io/grpc/binder/internal/BinderServer.java

```diff
@@ -161,7 +171,8 @@ public synchronized boolean handleTransaction(int code, Parcel parcel) {
                   .set(BinderTransport.REMOTE_UID, callingUid)
                   .set(BinderTransport.SERVER_AUTHORITY, listenAddress.getAuthority())
                   .set(BinderTransport.INBOUND_PARCELABLE_POLICY, inboundParcelablePolicy);
-          BinderTransportSecurity.attachAuthAttrs(attrsBuilder, callingUid, serverPolicyChecker);
+          BinderTransportSecurity.attachAuthAttrs(
+              attrsBuilder, callingUid, serverPolicyChecker, executor);
```

Comment:
Should we also wrap `executor` in `checkNotNull` since it's temporarily null?

That could highlight that it's nullable, but we expect a value by this point.

Reply:
IDK this doesn't seem much different than our use of `listener` below, which we simply assume isn't null (because started && !shutdown). But OK done

Reply:
TBH reading the `listener` could benefit from this. If we were using some static analysis like the nullness checker, it would complain about these unguarded accesses.

Worst case, we could detect and reject nulls from buggy code closer to their definition rather than later when they get use, possibly after getting passed from one class to the other.